apps/console/src/features/authentication-flow-builder/components/nodes/sign-in-box-node/sign-in-box-node.tsx

```diff
@@ -460,7 +460,6 @@ export const SignInBoxNode: FunctionComponent<SignInBoxNodePropsInterface> = (
             if (
                 [
                     IdentityProviderManagementConstants.TOTP_AUTHENTICATOR,
-                    IdentityProviderManagementConstants.SMS_OTP_AUTHENTICATOR,
```

Comment:
Only removed in Classic Editor: https://github.com/wso2/identity-apps/commit/749ee0d3921a77a9d34bf9d633f99d49673fb98d#diff-c07da02925a01edea431cc4d4d7cfddbd689dacbdb2132e87948e645cd9eef0aL166